sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,372 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureServiceClientOptions } from '@azure/core-arm';
+import * as coreHttp from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface AzureDigitalTwinsAPIOptions extends AzureServiceClientOptions {
+    // (undocumented)
+    baseUri?: string;
+}
+
+// @public
+export interface DigitalTwinModelsAddOptionalParams extends coreHttp.RequestOptionsBase {
+    models?: any[];
+}
+
+// @public
+export type DigitalTwinModelsAddResponse = Array<ModelData> & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: ModelData[];
+    };
+};
+
+// @public
+export type DigitalTwinModelsGetByIdResponse = ModelData & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: ModelData;
+    };
+};
+
+// @public
+export interface DigitalTwinModelsListOptionalParams extends coreHttp.RequestOptionsBase {
+    dependenciesFor?: string[];
+    digitalTwinModelsListOptions?: DigitalTwinModelsListOptions;
+    includeModelDefinition?: boolean;
+}
+
+// @public
+export interface DigitalTwinModelsListOptions {
+    maxItemCount?: number;
+}
+
+// @public
+export type DigitalTwinModelsListResponse = PagedModelDataCollection & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: PagedModelDataCollection;
+    };
+};
+
+// @public
+export interface DigitalTwinsAddHeaders {
+    eTag: string;
+}
+
+// @public
+export interface DigitalTwinsAddOptionalParams extends coreHttp.RequestOptionsBase {
+    ifNoneMatch?: IfNoneMatch;
+}
+
+// @public
+export interface DigitalTwinsAddRelationshipHeaders {
+    eTag: string;
+}
+
+// @public
+export interface DigitalTwinsAddRelationshipOptionalParams extends coreHttp.RequestOptionsBase {
+    ifNoneMatch?: IfNoneMatch1;
+    relationship?: any;
+}
+
+// @public
+export type DigitalTwinsAddRelationshipResponse = DigitalTwinsAddRelationshipHeaders & {
+    body: any;
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: DigitalTwinsAddRelationshipHeaders;
+        bodyAsText: string;
+        parsedBody: any;
+    };
+};
+
+// @public
+export type DigitalTwinsAddResponse = DigitalTwinsAddHeaders & {
+    body: any;
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: DigitalTwinsAddHeaders;
+        bodyAsText: string;
+        parsedBody: any;
+    };
+};
+
+// @public
+export class DigitalTwinsClient {
```

Comment:
One thing we should discuss is whether it makes sense to have separate clients for models, twins, components, and relationships.